frontend/src/lib/components/HTMLElementsDisplay/htmlElementsDisplayLogic.ts

```diff
@@ -57,6 +57,18 @@ export const htmlElementsDisplayLogic = kea<htmlElementsDisplayLogicType>([
                     ? preselect(providedElements, startingSelector)
                     : parsedSelectorsRaw,
         ],
+        parsedElements: [
+            (s) => [s.elements, s.visibleElements],
+            (elements: ElementType[], visibleElements: number) => {
```

Comment:
No need for types here because the types will be derived from the `(s) => [...]`.

Reply:
Resolved in 6dbb4dc